docs/linking/README.md

```diff
@@ -116,12 +122,15 @@ In the web authentication channel, the result is the PISP being instructed on
 a specific URL where this supposed user should be redirected. This URL should be
 a place where the user can prove their identity (e.g., by logging in).
 
-![Request consent](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/mojaloop/pisp/master/docs/linking/2-request-consent-web.puml)
+![Request consent](../out/linking/2-request-consent-web.svg)
 
 ### 1.3.2. OTP / SMS
 
-In the OTP authentication channel, the result is the PISP should ask user to enter OTP token received on his registered device (phone). **/sendOTP** is an example endpoint used by DFSP to send OTP token to user's device.
-![Request consent](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/mojaloop/pisp/master/docs/linking/2-request-consent-otp.puml)
+In the OTP authentication channel, the result is the PISP being instructed on
```

Comment:
I have a different understanding of the flow in the OTP Channel. In diagrams we can see that it is DFSP decision which channel will be used, then if it is OTP the DFSP use API call `/sendOTP` to schedule sending SMS. Is there a need to ask PISP to redirect the user to the DFSP login page to make additional authorization? If yes then there is a need to specify `authUri` in response and refactor in the sequence the place where `/sendOTP` is called.

Reply:
Oops, this was a copy paste error I think.

Reply:
ooh looks like it was there for a while... well spotted :)